test/non-functional/auth/my-privileges/common.ts

```diff
@@ -7,6 +7,20 @@ const cgrudPrivilege = [
   'UPDATE',
   'DELETE',
 ];
+
+export const cgrud_createComment_Privilege = [
+  ...cgrudPrivilege,
+  'CREATE_COMMENT',
+].sort();
+
+export const cgrud_createDiscussion_Privilege = [
+  'CREATE',
+  'GRANT',
+  ...readPrivilege,
+  'UPDATE',
+  'DELETE',
+  'CREATE_DISCUSSION',
+].sort();
```

Comment:
Please rename these to not use abbreviations (cgrud) but full meaning (i.e. create_grant_read_update_delete).